internals/webpack/webpack.base.babel.js

```diff
@@ -7,22 +7,25 @@ const webpack = require('webpack');
 
 module.exports = (options) => ({
   entry: options.entry,
-  output: { // Compile into js/build.js
-    path: path.resolve(__dirname, '../..', 'build'),
-    filename: '[name].js',
-    chunkFilename: '[name].chunk.js',
+  output: Object.assign({ // Compile into js/build.js
+    path: path.resolve(__dirname, '..', 'build'),
     publicPath: '/',
-  },
+  }, options.output), // Merge with env dependent settings
   module: {
     loaders: [{
       test: /\.js$/, // Transform all .js files required somewhere with Babel
       loader: 'babel',
-      exclude: path.join(__dirname, '../..', '/node_modules/'),
+      exclude: /node_modules/,
       query: options.babelQuery,
     }, {
       test: /\.css$/, // Transform all .css files required somewhere with PostCSS
+      exclude: /node_modules/,
       loader: options.cssLoaders,
     }, {
+      test: /\.css$/,
+      include: /node_modules/,
+      loaders: ['style-loader', 'css-loader'],
```

Comment:
Actually this is okay how it is. 

The point here is that they aren't being processed with css-modules and remain in global scope. And since we require these CSS files in our JS or CSS files, they will be a part of our compilation either way. So, no need for ExtractTextPlugin here.